sdk/tests/deqp/modules/shared/glsVertexArrayTests.js

```diff
@@ -553,14 +541,12 @@ goog.scope(function() {
 
         if (this.m_storage == glsVertexArrayTests.deArray.Storage.BUFFER) {
             this.m_ctx.bindBuffer(glsVertexArrayTests.ContextArray.targetToGL(target), this.m_glBuffer);
-            glsVertexArrayTests.GLU_EXPECT_NO_ERROR(this.m_ctx.getError(), "gl.bindBuffer()");
```

Comment:
Similar question as above regarding the removal of the error checks here. If the browsers are generating errors here, let's fix them rather than removing these checks.